MAUI/DataForm/getting-started.md

```diff
@@ -0,0 +1,342 @@
+---
+layout: post
+title: Getting Started with .NET MAUI DataForm control | Syncfusion
+description: Learn about getting started with Syncfusion .NET MAUI DataForm (SfDataForm) control and its basic features.
```

Comment:
Learn about getting started with Syncfusion .NET MAUI DataForm (SfDataForm) control and its basic features in mobile and desktop applications from a single shared codebase.

Reply:
title changed